docs/running-multiple-instances.md

```diff
@@ -0,0 +1,31 @@
+By default, Watchtower will clean up other instances and won't allow multiple instances monitoring the same containers. It is possible to override this behavior by defining a [scope](https://containrrr.github.io/watchtower/arguments/#filter_by_scope) to each running instance. 
+
+Notice that:
+- Multiple instances can run with the same scope;
+- An instance without a scope will clean up other running instances, even if they have a defined scope;
+
+To define an instance monitoring scope, use the `--scope` argument or the `WATCHTOWER_SCOPE` environment variable on startup and set the _com.centurylinklabs.watchtower.scope-uid_ label with the same value for the containers you want to include in this instance's scope (including the instance itself).
+
+For example, in a Docker Compose config file:
+
+```json
+version: '3'
+
+services:
+  app-monitored-by-watchtower:
+    image: myapps/monitored-by-watchtower
+    labels:
+      - "com.centurylinklabs.watchtower.enable=true"
+      - "com.centurylinklabs.watchtower.scope=myscope"
+
+  watchtower:
+    image: containrrr/watchtower
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    command: --debug --interval 30
+    labels:
+      - "com.centurylinklabs.watchtower.enable=false"
```

Comment:
What does this line do in this context? Disable self-update?

Reply:
Actually this whole example was pretty wrong:
- Port binding even though API mode is not enabled
- Use of `com.centurylinklabs.watchtower.enable` without passing `--label-enable`
- `--scope` is missing

I'm glad that you pointed it out so I could rewrite it =P